.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -5,19 +5,25 @@ body:
       - Search [open](https://github.com/godotengine/godot/issues) and [closed](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported. If you don't find a relevant match or if you're unsure, don't hesitate to **open a new issue**. The bugsquad will handle it from there if it's a duplicate.
-      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
+      - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html). Please always check if your issue is reproducible in the latest version – it may already have been fixed!
+      - If you use a custom build, please test if your issue is reproducible in official builds too. Likewise if you use any C++ modules, GDExtensions, or editor plugins, you should check if the bug is reproducible in a project without these.
 
-- type: input
+- type: textarea
   attributes:
-    label: Godot version
-    description: >
-      Specify the Godot version, including the Git commit hash if using a development or non-official build. The exact Godot version (including the commit hash) can be copied by clicking the version shown in the editor (bottom bar) or in the project manager (top bar).
-      If you use a custom build, please test if your issue is reproducible in official builds too.
-    placeholder: 3.5.stable, 4.0.dev [3041becc6]
+    label: Tested versions
+    description: |
+      To properly fix a bug, we need to identify when the bug was introduced in the engine, or if it was always present.
+      - Please specify the Godot version you found the issue in, including the **Git commit hash** if using a development or non-official build. The exact Godot version (including the commit hash) can be copied by clicking the version shown in the editor (bottom bar) or in the project manager (top bar).
+      - If you can, **please test earlier Godot versions** (previous stable branch, and development snapshots of the current feature release) and, if applicable, newer versions (development snapshots for the next feature release). Mention whether the bug is reproducible or not in the versions you tested. You can find all Godot releases on our [download archive](https://godotengine.org/download/archive/).
+      - The aim is for us to identify whether a bug is a **regression**, i.e. an issue that didn't exist in a previous version, but was introduced later on, breaking existing functionality. For example, if a bug is reproducible in 4.2.stable but not in 4.1.stable, we would like you to test intermediate 4.2 dev and beta snapshots to find which snapshot is the first one where the issue can be reproduced.
+    placeholder: |
+
+      - Reproducible in: 4.3.dev [d76c1d0e5], 4.2.stable, 4.2-dev5 and later 4.2 snapshots.
+      - Not reproducible in: 4.1.3-stable, 4.2-dev4 and earlier 4.2 snapshots.
```

Comment:
That empty first line in the placeholder was needed because for `textarea` fields, GitHub automatically adds a "Leave a comment" placeholder when you click the input field... and it overlaps the placeholder configured in the template :man_facepalming: 

See the screenshot in https://github.com/godotengine/godot/pull/85554#issuecomment-1838872518